### DISCUSSION TITLE
changement commande guildelder (changement d'elder)

### DISCUSSION THREAD

**Discussion by ghosts-arch** [Crownicles/Crownicles#1394] [Nouvelles idées]
Si un elder est déjà présent dans la guilde, changer le message actuel pour un message plus clair qui indique l'ancien et le nouveau elder.

**Comment by BastLast** [comment]
Screen du comportement actuel ?

**Reply by ghosts-arch** [reply]
![screen1](https://user-images.githubusercontent.com/112278606/192101706-8a1f99c3-9c8f-4a80-bac8-0d1f93117f4f.png)
![screen2](https://user-images.githubusercontent.com/112278606/192101709-6b654d94-2cee-4bf7-8312-a23f59b95c99.png)
